src/main/java/core/basesyntax/service/WriteDailyReportToFile.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.DataBase;
+import core.basesyntax.db.DataBaseCsvImpl;
+import java.util.List;
+
+public class WriteDailyReportToFile {

```

Comment:
```suggestion
public class FileWriterImpl implements FileWriter {
```
make this class responsible for just one task - writing data to a file